### ISSUE TITLE
Create a new contributor quickstart page to funnel potential new contributors to

### ISSUE THREAD

**Issue by mairin** [FNNDSC/website#6]
chrisproject.org needs a place for new contributors to learn about how they can get started with the project. Below is an initial wireframe idea of how this could look. Red Hat is going to be putting out some fresh media on ChRIS and would like to direct some traffic towards contributions, so we need a nice, easy-to-use landing page to drive those contributors to.

![Screenshot from 2020-08-06 09-16-27](https://user-images.githubusercontent.com/799683/89580015-b9fb2680-d802-11ea-8d4d-921628c788c5.png)

**Comment by mairin** [comment]
Updated snapshot so you can see WIP
![Screenshot from 2020-08-13 08-44-13](https://user-images.githubusercontent.com/799683/90190200-926d0680-dd8c-11ea-8864-be43340c92fe.png)

**Comment by junaruga** [comment]
As we discussed about the join-us page here in the Slack channel, I want to see some motivation parts inside chrisproject.org in the future.
https://fnndsc.slack.com/archives/C04FLFUA6/p1598535121028000

I think our demo site might be included somewhere as one of the steps in the funnel model designed page in the future.

I want to see the page view, unique user for each step and conversion rate between steps are measure if possible.
As you know, the tutorial is a very first part of the web site. It's more important to measure it than other pages.

**Comment by mairin** [comment]
@junaruga The tutorial isn't the first part, the front page is. But I think instrumenting the pages is a good idea. Can you open up a new issue for this?

**Comment by mairin** [comment]
(I'm going to close this issue now because the work is complete.)

**Comment by junaruga** [comment]
Yes, you are right. The front page is a very first part. It's likely that the tutorial is a very second part for the first visitors. Those pages are still in earlier timing than other pages.

> But I think instrumenting the pages is a good idea. Can you open up a new issue for this?

Sure. I will.

**Comment by junaruga** [comment]
Memo: Lisa has been working on this ticket, drafting the copy for the front page.

**Comment by mairin** [comment]
Lisa - I'm adding you to this ticket to review/edit the text when you're ready.

**Comment by mairin** [comment]
This page is now live at http://chrisproject.org/join-us
